Portfolio/index.css

```diff
@@ -1,15 +1,20 @@
-/* Write your code below for PORTFOLIO */
 * {
     margin: 0px;
     padding: 0px;
     font-family: "Revalia", sans-serif;

```

Comment:
There is an empty line (line 5) which should be removed for cleaner code structure.